robotpy_ext/common_drivers/pressure_sensors.py

```diff
@@ -0,0 +1,38 @@
+from wpilib import AnalogInput
+
+
+class REVAnalogPressureSensor():
+    '''
+    The REV Robotics Analog Pressure Sensor is a 5V sensor
+    that can measure pressures up to 200 PSI. It outputs an
+    analog voltage that is proportional to the measured pressure.
+
+    Pressure is derived from the following equation, taken from
+    the data sheet::
+
+        Pressure = 250 * (Vout / Vcc) -25
+
+    where Vout is the output voltage of the sensor, and Vcc is the input
+    voltage
+    '''
+
+    def __init__(self, channel, voltage_in=3.2):
+        self.sensor = AnalogInput(channel)
+        self.voltage_in = voltage_in
+
+    @property
+    def pressure(self):
+        v = max(self.sensor.getAverageVoltage(), 0.00001)
+        if hasattr(self, 'Vn'):
+            return (250 * (v / self.Vn)) - 25
+        return (250 * (v / self.voltage_in)) - 25
+
+    def normalize(self, known_pressure):
+        '''
+        Normalize calculated pressure to known pressure
```

Comment:
It's not exactly clear why this method exists? Maybe add something to the class docstring explaining how this class is intended to be used?